cloud-architecture/oracle-apps-erp/cyber-recovery-solution/files/Cyber-Recovery-Solution.md

```diff
@@ -0,0 +1,257 @@
+# Introduction
```

Comment:
Maybe we need a license comment here - the script could find its license. @oheimburger what do you think? Do we need a license in a header comment or can we ignore the script?